assets/js/pages/HostDetailsPage/HostDetailsPage.jsx

```diff
@@ -92,6 +103,9 @@ function HostDetailsPage() {
       selectedChecks={hostSelectedChecks}
       slesSubscriptions={host.sles_subscriptions}
       catalog={catalog}
+      relevantPatches={numRelevantPatches}
+      upgradablePackages={numUpgradablePackages}
+      softwareUpdatesLoading={softwareUpdatesLoading}
```

Comment:
Maybe we wanna add a tooltip text as a prop here? I really don't know, and I really don't wanna block this PR just for this nit

Reply:
Done, do you think it's worth testing the tooltip behaviour from `HostDetails.test.jsx`, as it's already being tested in `AvailableSoftwareUpdates.test.jsx`... :thinking:

Reply:
No, I think we can iterate over that, for example in the future we can have a selector to determine tooltip's text